Validation/FutureDateAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+
+namespace Airport_Ticket_Booking.Validation
+{
+    public class FutureDateAttribute : ValidationAttribute
+    {
+        public override bool IsValid(object value)
+        {
+            if (value is DateTime date)
+            {
+                return date > DateTime.Now;
+            }
+            return false;
+        }
+    }
+}
```

Comment:
1. use scoped name space (please apply this to all other classes). 
2. add nullable operator for the input parameter so it can match the overridden method signature.
```suggestion
namespace Airport_Ticket_Booking.Validation;

public class FutureDateAttribute : ValidationAttribute
{
    public override bool IsValid(object? value)
    {
        if (value is DateTime date)
        {
            return date > DateTime.Now;
        }
        return false;
    }
}
```